pycaret/utils/time_series/__init__.py

```diff
@@ -118,7 +118,7 @@ def get_diffs(
 
 def _get_diff_name_list(
     data: pd.Series, data_name: Optional[str] = None, data_kwargs: Optional[Dict] = None
-) -> Tuple[List[pd.Series], List[str]]:
+) -> Tuple[List[pd.Series], List[Optional[str]]]:
```

Comment:
I think it will always have data in it. Will it ever be optional?

Reply:
@ngupta23 
Thank you for checking!

> I think it will always have data in it. Will it ever be optional?

The type of the argument `data_name` of `_get_diff_name_list` is `Optional[str]`, which defaults to `None`. Therefore, in the following process, the type will be as I have fixed it.

```python
    else:
        # Issue with reconciliation of orders and diffs
        diff_list = [data]
        name_list = [data_name]

    return diff_list, name_list
```

Specifically, the code says that `name_list` may have the value `[None]`.
There may be a mistake in my proposed type or logic. Please check here.